### ISSUE TITLE
Can fail a11y testing with axe-core 4.11.3

### ISSUE THREAD

**Issue by jrjohnson** [ember-a11y/ember-a11y-refocus#437]
With the release of axe-core [4.11.3](https://github.com/dequelabs/axe-core/releases/tag/v4.11.3) I'm getting failures out of ember-a11y-testing `The skip-link target should exist and be focusable` when I have the `<NavigationNarrator />` in a component and `<main id="main">` is elsewhere. This seems right there is no target in the DOM for this test so it's a legit failure. Was just surprising to see it from a patch release in axe-core.

More informational than a bug I think, I think the fix is to skip this test in components.

```
await a11yAudit(this.element, {
  rules: {
    'skip-link': {
      enabled: false,
    },
  },
});
```
